guides/v2.2/contributor-guide/contributing.md

```diff
@@ -235,15 +191,15 @@ As a best practice, we recommend creating back-ports and up-ports for your code
 
 **Back-ports** (or Backports) contribute your code and fixes to a lower release line. You want to create a back-port if the issue exists in a lower release line. If you fixed an issue in a release line and there is a supported lower version, create a pull request to that lower release line to address the issue. For example, you may have contributed a fix to 2.3 and back-port to 2.2 and 2.1.
 
-**Up-ports** (or Forwardports) contribute your code and fixes to a higher release line. You want to create an up-port if the issue exists in a higher release line. If you have an issue fixed in a non-latest release line, create a pull request to the latest branch in order to address that issue in an upcoming minor release. For example, you may have contributed a fix to 2.1 and up-port to 2.2 and 2.3. We recommend contributors create an up-port for every pull request delivered to a lower release line.
+**Up-ports** (or Forwardports) contribute your code and fixes to a higher release line. You want to create an up-port if the issue exists in a higher release line. If you have an issue fixed in a non-latest release line, create a pull request to the latest branch in order to address that issue in an upcoming minor release. For example, you may have contributed a fix to 2.2 and up-port to 2.3. We recommend contributors create an up-port for every pull request delivered to a lower release line.
```

Comment:
```suggestion
**Up-ports** (or Forwardports) contribute your code and fixes to a higher release line. Create an up-port if the issue exists in a higher release line. If you have an issue fixed in a non-latest release line, create a pull request to the latest branch in order to address that issue in an upcoming minor release. For example, you may have contributed a fix to 2.2 and an up-port to 2.3. We recommend contributors create an up-port for every pull request delivered to a lower release line.
```

Reply:
Same suggestion here -- "newer release" instead of "higher release" .  

Also, "if the issue exists" ==>  "If the issue affects a newer release line"

If you have an issue fixed in a non-latest release line ==> If you have fixed an issue in an earlier release line, create a pull request to the latest release branch to port the change to the upcoming minor release.

For example, when you create a fix to 2.2, create a pull request to up-port the fix to the 2.3 release branch.  We recommend that contributors create an up-port for every pull request delivered to an older release line.